docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Caffe.md

```diff
@@ -45,13 +44,13 @@ Caffe*-specific parameters:
                         attributes without flattening nested parameters.
 ```
 
-### Command-Line Interface (CLI) Examples Using Caffe\*-Specific Parameters
+## Command-Line Interface (CLI) Examples Using Caffe-Specific Parameters
 
-* Launching the Model Optimizer for the [bvlc_alexnet.caffemodel](https://github.com/BVLC/caffe/tree/master/models/bvlc_alexnet) with a specified `prototxt` file. This is needed when the name of the Caffe\* model and the `.prototxt` file are different or are placed in different directories. Otherwise, it is enough to provide only the path to the input `model.caffemodel` file.
+* Launching the Model Optimizer for the [bvlc_alexnet.caffemodel](https://github.com/BVLC/caffe/tree/master/models/bvlc_alexnet) with a specified `prototxt` file. This is needed when the name of the Caffe model and the `.prototxt` file are different or are placed in different directories. Otherwise, it is enough to provide only a path to the input `model.caffemodel` file.
```

Comment:
```suggestion
* Launching Model Optimizer for [bvlc_alexnet.caffemodel](https://github.com/BVLC/caffe/tree/master/models/bvlc_alexnet) with a specified `prototxt` file.
   This is needed when the name of the Caffe model and the `.prototxt` file are different or are placed in different directories. Otherwise, it is enough to provide only the path to the input `model.caffemodel` file.
```